examples/vaadin-lts-v08-simple/README_pt.adoc

```diff
@@ -0,0 +1,56 @@
+== Vaadin V8 (LTS) - WebApp Simples em Java
```

Comment:
Hi @marcoantoniobferreira , thanks for the PR.
I'll suggest adding the document headers to define jbake metadata and index-group.
Check for instance: https://github.com/apache/tomee/blame/master/examples/mp-metrics-gauge/README.adoc#L3-L5

Reply:
Hi @cesarhernandezgt thanks for review. In this case, which is the correct index-group?

Reply:
Currently, appears on Uknown section:
![image](https://user-images.githubusercontent.com/1454523/64400604-394c0d00-d02a-11e9-8f9f-f7afbbedf964.png)

IMHO, I think it should be under Frameworks:
![image](https://user-images.githubusercontent.com/1454523/64400626-4cf77380-d02a-11e9-8b03-2e90c17f545f.png)

If you want to test the end result of the index-group, you will need to run the website locally on your machine to make sure it appear correctly: https://github.com/apache/tomee-site-generator/